### PR TITLE
chore: drop sqlite3 (DWH)

### PR DESCRIPTION
**Pull request by oopcode**
This PR removes `sqlite3` support from DWH.
Postgres for tests runs in a docker container.
A new target (`make test/lite`) is added to `Makefile`, it runs tests **without** DWH tests. Old target `make test` sets up a docker container with `postgres` and runs **all** tests (including DWH).

### CODE REVIEW

**Comment by oopcode** [comment]
dear @sokel, 

1. please learn to stay away from [WIP] pull requests, 
2. now there's two targets, `make test` (runs all tests, will fail on a machine with no `postgres`) and `make test/unit` which only runs tests that do not need any external entities to be set up.

File: insonmnia/dwh/server_test.go [hunk 1]
```diff
@@ -16,49 +15,114 @@ import (
 	bch "github.com/sonm-io/core/blockchain"
 	pb "github.com/sonm-io/core/proto"
 	"github.com/stretchr/testify/require"
-)
-
-const (
-	testDBPath        = "test_dwh.db"
-	testMonitorDBPath = "test_monitor_dwh.db"
+	"golang.org/x/net/context"
```
[the remark this answers is not in the document]
Comment (oopcode) [reply insonmnia/dwh/server_test.go]: fixed!

File: insonmnia/dwh/server_test.go [hunk 2]
```diff
@@ -18,47 +26,193 @@ import (
...
+		if _, err := db.Exec(fmt.Sprintf("DROP DATABASE IF EXISTS %s", dbName)); err != nil {
+			return fmt.Errorf("failed to drop database: %s", err)
+		}
+	}
+
+	return nil
+}
+
+func getConnString(database, user, password string) string {
+	return fmt.Sprintf("postgresql://localhost:15432/%s?user=%s&password=%s&sslmode=disable", database, user, password)
```
Comment (sshaman1101) [inline insonmnia/dwh/server_test.go]: maybe will be better to not hardcode port number and use a value from constants

File: insonmnia/dwh/server_test.go [hunk 3]
```diff
@@ -18,47 +26,193 @@ import (
...
+
+func setupDB() error {
+	db, err := sql.Open("postgres", serviceConnString)
+	if err != nil {
+		return fmt.Errorf("failed to connect to template1: %s", err)
+	}
+	defer db.Close()
+
+	for _, dbName := range []string{globalDBName, monitorDBName} {
+		if _, err := db.Exec(fmt.Sprintf("DROP DATABASE IF EXISTS %s", dbName)); err != nil {
```
Comment (sshaman1101) [inline insonmnia/dwh/server_test.go]: For those who'll review later: prepared statements will be more clear here, but the postgresql [does not support](https://github.com/lib/pq/issues/694) it for services statements.

File: insonmnia/dwh/server_test.go (hunk 2 above)
Comment (oopcode) [reply insonmnia/dwh/server_test.go]: will do that in the next round of refactoring

**Pull request merged**